Add 64 bits optionnal check for DXR  wizard (for DXR player)

remi-chapelain opened the pull request:
### Purpose of this PR
RTX effects needs to have a 64 bits architecture for them to work in player but it was not mentioned anywhere. 
[This has just been added to the doc](https://github.com/Unity-Technologies/Graphics/pull/1631) but since users don't read the doc, adding it to the optional check in the DXR wizard would be a great addition to what's already there. 

![image](https://user-images.githubusercontent.com/57442369/90534831-7c8b8700-e17a-11ea-84b8-905b232bc4d1.png)

![image](https://user-images.githubusercontent.com/57442369/90534958-ac3a8f00-e17a-11ea-9277-109fbe0bb9c3.png)

---
### Testing status
Not a lot to test there. 
I just tried to change architecture to see if that is affecting correctly the check

### Comments to reviewers
Just want to make sure that it makes sense and I did not make obvious mistakes.

iM0ve commented:
Good addition, adds an informative text as in the images below for DXR.

Tested:
* The warning is visible when 32 bit is selected
* Warning is not visible in other non DXR menus
* Warning turns to green checkbox when build is set to 64 

Suggestion:
I think this option would benefit from having Fix button. Is there a downside why user would want to keep 32 bits in his DXR project? If 32 vs 64 is not an option, because 32 just doesnt work we should include this fix in the "Fix all" list if possible.

remi-chapelain commented:
Following @iM0ve comment, I have changed the check from optional to required and added a fix button for it. 
I've check the fix button on a fresh new project and also if it works with the top "fix all" button.

![image](https://user-images.githubusercontent.com/57442369/90755494-74972880-e2db-11ea-9651-c187b173d28e.png)

iM0ve reviewed (changes requested):
Can we address this warning?

![image](https://user-images.githubusercontent.com/36502659/90761917-cf854b80-e2ec-11ea-8ebb-dbc88ab75abc.png)

remi-chapelain commented:
Obsolete function request fixed !

iM0ve reviewed (approved):
All seems good now

RSlysz commented:
> 
> 
> Following @iM0ve comment, I have changed the check from optional to required and added a fix button for it.
> I've check the fix button on a fresh new project and also if it works with the top "fix all" button.
> 
> ![image](https://user-images.githubusercontent.com/57442369/90755494-74972880-e2db-11ea-9651-c187b173d28e.png)

You just broke the resources group :O Please put the "Architecture 64 bits" outside of "DXR resources" (before or after but not inside especially if not indented.

RSlysz commented on com.unity.render-pipelines.high-definition/Editor/Wizard/HDWizard.Configuration.cs:
```diff
@@ -174,6 +174,8 @@ Entry[] entries
                         new Entry(InclusiveScope.DXR, Style.dxrStaticBatching, IsDXRStaticBatchingCorrect, FixDXRStaticBatching),
                         new Entry(InclusiveScope.DXR, Style.dxrActivated, IsDXRActivationCorrect, FixDXRActivation),
                         new Entry(InclusiveScope.DXR, Style.dxrResources, IsDXRAssetCorrect, FixDXRAsset),
+                        new Entry(InclusiveScope.DXR, Style.dxr64bits, IsArchitecture64Bits, FixArchitecture64Bits),
```
Move this line one line above to not break grouping of "DXR Resources"

remi-chapelain commented:
> You just broke the resources group :O Please put the "Architecture 64 bits" outside of "DXR resources" (before or after but not inside especially if not indented.

Did not realized thoses were grouped ! I moved the 64 bits thingy above ! Thanks for pointing it out.

anisunity reviewed (approved):
lgtm
